### DISCUSSION TITLE
Exclude Monero blockchain from backup

### DISCUSSION THREAD

**Discussion by shawn-digitalpoint** [btcpayserver/btcpayserver#3201] [Ideas / Feature Requests]
Similar to how Bitcoin and Litecoin blockchain are excluded when using the backup.sh script, it wouldn't be a terrible idea to exclude Monero's blockchain.

Instead of this:
`tar --exclude="$backup_dir/*" --exclude="$volumes_dir/generated_bitcoin_datadir/*" --exclude="$volumes_dir/generated_litecoin_datadir/*" --exclude="$volumes_dir/**/logs/*" -cvzf $backup_path $dbdump_path $volumes_dir`

...it could be this:
`tar --exclude="$backup_dir/*" --exclude="$volumes_dir/generated_bitcoin_datadir/*" --exclude="$volumes_dir/generated_litecoin_datadir/*" --exclude="$volumes_dir/generated_xmr_data/*" --exclude="$volumes_dir/**/logs/*" -cvzf $backup_path $dbdump_path $volumes_dir`

Figured out that this was the cause of TERRIBLY slow backups (and extended btcpayserver downtime while backup was happening).

**Comment by ksdhans** [comment]
I second this. The monero blockchain is ~127GB and growing, and there's no need to include it in the backup

**Comment by bolatovumar** [comment]
@shawn-digitalpoint @ksdhans I created a PR which should address this issue here: https://github.com/btcpayserver/btcpayserver-docker/pull/648
